src/components/TodoList.tsx

```diff
@@ -0,0 +1,117 @@
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todos: Todo[] | null,
+  filteringMode: string;
+}
+
+let filteredTodos: Todo[] | null = [];
```

Comment:
This can be just an empty array if there's nothing in it, you don't need to set it to null. If you want to perform a check if this exist use `array.length`
```suggestion
let filteredTodos: Todo[] = [];
```